### PR TITLE
827 - Fixed busy indicator was showing offset inside modal

### LINKED ISSUE #827: Busy Indicator is pushed way down when in a modal dialog
**Describe the bug**
When using a busy indicator on a modal it can be pushed far down out of view

**To Reproduce**
[soho-busyindicator-in-modal-dialog.zip](https://github.com/infor-design/enterprise/files/2426153/soho-busyindicator-in-modal-dialog.zip)
1. Unzip the sample
1. Bring up a modal dialog.
1. Click on a dialog button to activate the busy indicator.
1. The busy indicator bars are behind the dialog.  However, if the overlay is not transparent, it displays over the dialog.

**Expected behavior**
Busy indicator in the middle and visible.

**Screenshots**
busyindicator

**Platform**
- All

### PR DESCRIPTION
**Pull request by deep7102**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
Busy indicator pushed far down out of view, when it was using inside the modal.

**Related github/jira issue (required)**:
Closes #827

**Steps necessary to review your pull request (required)**:
http://localhost:4000/components/busyindicator/test-inside-modal.html
- Open above link
- Click on button "Show Modal" to bring up the modal dialog
- Click on button "Show Busy Indicator" to activate the busy indicator
- See the Busy Indicator should show fine

### CODE REVIEW

**Comment by deep7102** [comment]
Yes! agreed @EdwardCoyle customizable overlay or bar colors, but currently we do not have personalize setting for it would be nice to have one.

**Comment by tmcconechy** [comment]
@deep7102 I think @EdwardCoyle is right. What you can do is just add a style under .modal so that `.busy.overlay {` background color is white and not #f0f0f0. Something like...

```
.modal .busy.overlay {
  background-color: $theme-color-palette-white; 
}
```

File: app/views/components/busyindicator/test-inside-modal.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+		<button class="btn-secondary" type="button" id="add-context">Show Modal</button>
+		<br/><br/>
+
+		<!-- Modal Example -->
+		<div id="modal-add-context" class="hidden busy" data-options="{ 'displayDelay': 100, 'timeToComplete': 5000 }">
+			<div style="padding: 10px 0;">
+	      <button class="btn-secondary" type="button" id="show-busy-indicator">Show Busy Indicator</button>
+	    </div>
+		  <p>
```
Comment (tmcconechy) [inline app/views/components/busyindicator/test-inside-modal.html]: Mixed tabs and spaces in this file.

**Comment by deep7102** [comment]
Updated now, overlay color and removed tabs

**Comment by janahintal** [comment]
QA Passed. Working well on all Devices, Browser and OS. Will move to done now. 
Tested in: http://master-enterprise.demo.design.infor.com/components/busyindicator/test-inside-modal.html & http://localhost:4000/components/busyindicator/test-inside-modal.html

**Pull request merged**